src/magicbox/MagicBox.ts

```diff
@@ -166,19 +165,28 @@ export class MagicBoxInstance {
     };
 
     this.inputManager.onkeyup = (key: number) => {
-      if (key == 38) {
-        // Up
-        this.onmove && this.onmove();
-        this.focusOnSuggestion(this.suggestionsManager.moveUp());
-        this.onchange && this.onchange();
-      } else if (key == 40) {
-        // Down
-        this.onmove && this.onmove();
-        this.focusOnSuggestion(this.suggestionsManager.moveDown());
-        this.onchange && this.onchange();
-      } else {
-        return true;
+      let selected: Suggestion | HTMLElement = null;
```

Comment:
I would prefer to avoid union types of this kind. They make code complicated and require downstream checks on the instance type as you have on lines 186. I will leave some comments about this in the SuggestionsManager.ts file.